tests/mmrelay/test_matrix_utils.py

```
@@ -0,0 +1,1037 @@
+        frobenius_norm,
+        condition_number,
+        solve_linear_system,
+        decompose_lu,
+        decompose_qr,
+        is_positive_definite,
+        matrix_power,
+        kronecker_product,
+        vectorize_matrix,
+        reshape_matrix,
+    )
+        is_singular,
```

Pylint (3.3.7)

[error] Parsing failed: 'unexpected indent (tests.mmrelay.test_matrix_utils, line 45)'

(E0001)